.clang-format

```diff
@@ -1,8 +1,9 @@
 ---
-#Based on clang-format 11.0.0
+# Based on clang-format 11.0.0
 Language:        Cpp
 BasedOnStyle:    google
-Standard:        c++17
+# Standard:        c++17
+Standard:        Auto
```

Comment:
Why change from c++17 standard?  Auto picks detects standard based on input.   Not sure which is best in our case.

Reply:
I used merlin during development and it threw an error when using c++17. So I changed to Auto. Maybe my VS Code setup has an issue. Let's discuss it in this afternoon meeting.

Reply:
I think the root of this 'issue' is the version of clang-format.  clang-format-6 only has support for Cpp11.  https://releases.llvm.org/6.0.0/tools/clang/docs/ClangFormatStyleOptions.html

Reply:
Keep as is